code/modules/spells/spell_types/spacetime_distortion.dm

```diff
@@ -62,6 +62,7 @@
 	desc = "A distortion in spacetime. You can hear faint music..."
 	icon_state = "wave1"
 	color = "#8A2BE2"
+	anchored = 1
```

Comment:
@PKPenguin321 if it's not a huge bother to make another commit, define this ``anchored = 1`` in the parent type ``/obj/effect/cross_action``